ptbr/config_cli_old.py

```
@@ -0,0 +1,842 @@
+            report.add_warning(
+                f"{section_prefix}.{key}",
+                f"Unknown field (will be ignored): {key!r}",
+            )
+
+    return result
+
+
+def _validate_cross_constraints(
+    gliner_data: Dict[str, Any],
+    method: str,
+    full_or_lora: str,
```

Ruff (0.15.1)

[warning] Unused function argument: `full_or_lora`

(ARG001)